src/app/utils/utils.js

```diff
@@ -368,6 +370,60 @@ function getAggregatedElectronicResources(items = []) {
   return _flatten(electronicResources);
 }
 
+/**
+ * Return a list of non Aeon Link Electronic Resouces from given list
+ * @param {array} resources Resources[ ]
+ * @param {array} items Items[ ]
+ * @return {array}
+ */
+function pluckAeonLinksFromResource(resources = [], items = []) {
```

Comment:
Related to above comment on documentation, could these param names reflect that they're actually the same type of thing - just different categories? (e.g. Call them `electronicItems` and `items`) Just a suggestion; I struggled to understand the logic before realizing how `items` and `resources` related to each other and I wonder if naming could help.

Reply:
Refer to above comment